README.md

```diff
@@ -144,20 +144,24 @@ To set your own `AppData`, change `REACT_APP_FULL_APP_DATA_<environment>` enviro
 
 ### Supported networks
 
-You can change the supported networks and their RPC endpoint.
-
-To have the interface default to a different network when a wallet is not connected:
-
-1. Change `REACT_APP_NETWORK_ID` to `"{YOUR_NETWORK_ID}"`. This will be your default network id
-2. Define your own list of supported networks:
+You should set your own RPC endpoints by defining the following environment variables:
 
 ```ini
-REACT_APP_SUPPORTED_CHAIN_IDS="1,100,5"
```

Comment:
There are more places where this is still defined https://github.com/search?q=repo%3Acowprotocol%2Fcowswap+REACT_APP_SUPPORTED_CHAIN_IDS&type=code

Reply:
done in a follow up PR https://github.com/cowprotocol/cowswap/pull/3161/commits/37c6e4b33bb27da0761f8df6e2ac784157ea2ed4